gen3users/validation.py

```diff
@@ -238,3 +263,26 @@ def validate_users(user_yaml_dict):
         ), 'Policies {} for user "{}" are not defined in list of policies'.format(
             invalid_policies, user_email
         )
+
+        # check resource paths in "users.projects.resource" are valid
+        # given "rbac.resources" resource tree
+        for project in user_access.get("projects", {}):
+            if "resource" in project:
+                assert project["resource"].startswith(
+                    "/"
+                ), 'Resource path "{}" in project "{}" for user "{}" should start with a "/"'.format(
+                    project["resource"], project["auth_id"], user_email
+                )
+
+                assert (
+                    project["resource"] in existing_resources
+                ), 'Resource "{}" in project "{}" for user "{}" is not defined in resources tree'.format(
+                    project["resource"], project["auth_id"], user_email
+                )
+
+                parts = project["resource"].split("/")
+                assert (
```

Comment:
I don't think this is an actual requirement, just a convention we're using.. I guess it doesn't hurt to force this tho, since ideally this resource field is going away